### PR TITLE
[#163591935] Deadlines with load of the past data

### PR DESCRIPTION
**Pull request by fpersico**
Deadlines

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#163591935](https://www.pivotaltracker.com/story/show/163591935): Come CIT voglio poter vedere filtrare, fra i messaggi, solo quelli che hanno una scadenza

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/components/messages/MessageAgenda.tsx [hunk 1]
```diff
@@ -30,49 +31,160 @@ const keyExtractor = (_: MessageWithContentAndDueDatePO) => _.id;
 
 const ItemSeparatorComponent = () => <View style={styles.itemSeparator} />;
 
+const SECTION_HEADER_HEIGHT = 50;
```
Comment (cloudify) [inline ts/components/messages/MessageAgenda.tsx]: Please add a comment that explain the purpose of this

File: ts/components/messages/MessageAgenda.tsx [hunk 2]
```diff
@@ -30,49 +31,160 @@ const keyExtractor = (_: MessageWithContentAndDueDatePO) => _.id;
...
+
+const ITEM_SEPARATOR_HEIGHT = 1;
```
Comment (cloudify) [inline ts/components/messages/MessageAgenda.tsx]: Please add a comment that explain the purpose of this

File: ts/components/messages/MessageAgenda.tsx [hunk 3]
```diff
@@ -30,49 +31,160 @@ const keyExtractor = (_: MessageWithContentAndDueDatePO) => _.id;
...
+
+const ITEM_HEIGHT = 100;
```
Comment (cloudify) [inline ts/components/messages/MessageAgenda.tsx]: Please add a comment that explain the purpose of this

File: ts/components/messages/MessageAgenda.tsx [hunk 4]
```diff
@@ -30,49 +31,160 @@ const keyExtractor = (_: MessageWithContentAndDueDatePO) => _.id;
...
+        offset,
+        index
+      });
+
+      offset += heightAndOffset;
+      index++;
+    });
+
+    // Push the info about the SECTION_FOOTER cell.
+    // NOTE: Also if not rendered the VirtualizedSectionList component create a cell.
```
Comment (cloudify) [inline ts/components/messages/MessageAgenda.tsx]:
```suggestion
    // NOTE: VirtualizedSectionList component creates a cell instance even when not rendered.
```

**Comment by cloudify** [comment]
This pull request **introduces 1 alert** when merging eaa6fd7a75acb54c9ef8fbf7eb6532b55bf55cb5 into e2978374ef84cb2bae289debbeb401d705a7364b - [view on LGTM.com](https://lgtm.com/projects/g/teamdigitale/io-app/rev/pr-5cc2365e3b15ae53ff271e69ff98fe1e90d64b4b)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fpersico** [comment]
About the LGTM comment: currentTimeLimit is used here https://github.com/teamdigitale/io-app/pull/929/files#diff-e0ab66df261825651fa0d7edccd8b46aR174

**Comment by cloudify** [comment]
This pull request **introduces 1 alert** when merging ff957e02babf9f37e8b9c29509969925af1b3b8e into e2978374ef84cb2bae289debbeb401d705a7364b - [view on LGTM.com](https://lgtm.com/projects/g/teamdigitale/io-app/rev/pr-9d604e69f0c1f1e9b044b1a2981ed3ab53e58f3e)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by cloudify** [comment]
This pull request **introduces 1 alert** when merging 3399adcda9ad2552275c0cc39534a2d0b3c45545 into e2978374ef84cb2bae289debbeb401d705a7364b - [view on LGTM.com](https://lgtm.com/projects/g/teamdigitale/io-app/rev/pr-13dd6ba5971d635874adf71375b5d426b94e121b)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by cloudify** [comment]
This pull request **introduces 1 alert** when merging 01bad98ec7241cdfd76a8768e1aa5e41d142aaff into 10d496643d26d04a499cee8c6ce2bb73e8bdc662 - [view on LGTM.com](https://lgtm.com/projects/g/teamdigitale/io-app/rev/pr-7146de0d87a2d647d4649b5a8ead8a0236e4d0db)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request closed without merging**